setup.py

```diff
@@ -6,36 +6,6 @@
 
 import os, glob, numpy, subprocess, sys
 
-PY2 = sys.version_info.major < 3
```

Comment:
this change basically requires Python 3? I'm personally not opposed but I think this repo does need ongoing python 2 support. @plaplant can you comment?